components/linking_landing_page/linking_landing_page.tsx

```diff
@@ -0,0 +1,392 @@
+                    defaultMessage='You can view it in Mattermost mobile app or continue in the web browser.'
+                />
+            );
+        }
+
+        return (
+            <div className='get-app__launching'>
+                <FormattedMessage
+                    id='get_app.launching'
+                    defaultMessage='Where would you like to view this?'
+                />
+                <div className='get-app__alternative'>
+                    {viewApp}
+                </div>
+            </div>
+        );
+    }
+
+    renderDialogBody = () => {
+        if (this.state.redirectPage) {
+            return (
+                <div className='get-app__dialog-body'>
+                    {this.renderDialogHeader()}
+                    {this.renderDownloadLinkSection()}
+                </div>
+            );
+        }
+
+        return (
+            <div className='get-app__dialog-body'>
+                {this.renderDialogHeader()}
+                <div className='get-app__buttons'>
+                    <div className='get-app__status'>
+                        {this.renderGoNativeAppMessage()}
+                    </div>
+                    <div className='get-app__status'>
+                        <a
+                            href='#'
+                            onClick={this.openInBrowser}
+                            className='btn btn-default btn-lg get-app__continue'
+                        >
+                            <FormattedMessage
+                                id='get_app.continueToBrowser'
+                                defaultMessage='View In Browser'
+                            />
+                        </a>
+                    </div>
+                </div>
+                <div className='get-app__preference'>
+                    <button
+                        className={`get-app__checkbox ${this.state.rememberChecked ? 'checked' : ''}`}
+                        onClick={this.handleChecked}
+                    >
+                        {this.renderCheckboxIcon()}
+                    </button>
+                    <FormattedMessage
+                        id='get_app.rememberMyPreference'
+                        defaultMessage='Remember my preference'
+                    />
+                </div>
+                {this.renderDownloadLinkSection()}
+            </div>
+        );
+    }
+
+    render() {
+        const isMobile = UserAgent.isMobile();
+
+        return (
+            <div className='get-app'>
+                <div className='get-app__header'>
+                    <img
+                        src={MattermostLogoSvg}
+                        className='get-app__logo'
+                    />
+                </div>
+                <div className='get-app__dialog'>
+                    <div
+                        className={`get-app__graphic ${isMobile ? 'mobile' : ''}`}
+                        style={{backgroundColor: this.props.backgroundColor}}
```

Comment:
Could we use CSS variables for this, or do those not exist at this point?

Reply:
I believe we can actually, I'll look into that.

Reply:
Done!